go-apps/meep-rnis/sbi/rnis-sbi.go

```diff
@@ -230,19 +243,24 @@ func processActiveScenarioUpdate() {
 						if domain.CellularDomainConfig != nil {
 							mnc = domain.CellularDomainConfig.Mnc
 							mcc = domain.CellularDomainConfig.Mcc
+							cellId = domain.CellularDomainConfig.DefaultCellId
 						}
-						if nl.Poa4GConfig != nil {
-							if nl.Poa4GConfig.CellId != "" {
-								cellId = nl.Poa4GConfig.CellId
-							} else {
-								cellId = domain.CellularDomainConfig.DefaultCellId
+						switch nl.Type_ {
+						case mod.NodeTypePoa4G:
+							if nl.Poa4GConfig != nil {
+								if nl.Poa4GConfig.CellId != "" {
+									cellId = nl.Poa4GConfig.CellId
+								}
 							}
-						} else {
-							if domain.CellularDomainConfig != nil {
-								cellId = domain.CellularDomainConfig.DefaultCellId
+						case mod.NodeTypePoa5G:
+							if nl.Poa5GConfig != nil {
+								if nl.Poa5GConfig.CellId != "" {
+									cellId = nl.Poa5GConfig.CellId
+								}
 							}
+						case mod.NodeTypePoaWifi:
```

Comment:
Use "default:" for the unsupported technology types. This way we will not have to re-work this if we add for say mmWave

Reply:
default added

Reply:
- Remove `case PoaWifi` as it does the same as `default` + RNIS should only care about 3GPP tech.
- Add this comment in default: `// Empty cellId is ignored by RNIS`